.github/workflows/deploy-examples.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
     - name: Build examples
       run: |
-        SOURCE_CODE_BASE="$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
+        SOURCE_CODE_BASE="$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/tree/$GITHUB_REF_NAME"
```

Comment:
Hmm, tree or blob?

Reply:
Auto-response: both work

Reply:
it's the same
https://github.com/alijdens/spark-board/tree/main/tests/examples/mix.py